### ISSUE TITLE
Please include requirements.txt in PyPi source distribution

### ISSUE THREAD

**Issue by tomfitzhenry** [alexdelorenzo/aiopath#17]
I'm trying to package aiopath for NixOS. This requires the source distribution to include `requirements.txt` (similar to https://github.com/nix-community/pypi2nix/issues/456 ).

aiopath's Pypi distribution does not include requirements.txt
```
$ curl -s 'https://files.pythonhosted.org/packages/4e/ab/04f87a414f227d92bc1278202356aea71ebdb1d75f31f16b1ad057c70c44/aiopath-0.5.11.tar.gz' | tar ztf -
aiopath-0.5.11/
aiopath-0.5.11/LICENSE
aiopath-0.5.11/PKG-INFO
aiopath-0.5.11/README.md
aiopath-0.5.11/aiopath/
aiopath-0.5.11/aiopath/__init__.py
aiopath-0.5.11/aiopath/flavours.py
aiopath-0.5.11/aiopath/handle.py
aiopath-0.5.11/aiopath/path.py
aiopath-0.5.11/aiopath/scandir.py
aiopath-0.5.11/aiopath/selectors.py
aiopath-0.5.11/aiopath/types.py
aiopath-0.5.11/aiopath/wrap.py
aiopath-0.5.11/aiopath.egg-info/
aiopath-0.5.11/aiopath.egg-info/PKG-INFO
aiopath-0.5.11/aiopath.egg-info/SOURCES.txt
aiopath-0.5.11/aiopath.egg-info/dependency_links.txt
aiopath-0.5.11/aiopath.egg-info/requires.txt
aiopath-0.5.11/aiopath.egg-info/top_level.txt
aiopath-0.5.11/aiopath.egg-info/zip-safe
aiopath-0.5.11/pyproject.toml
aiopath-0.5.11/setup.cfg
aiopath-0.5.11/setup.py
```

You mpris_server does, for example:
```
$ curl -s 'https://files.pythonhosted.org/packages/9c/0c/bd61ff9584f4597828bee8538a68be08cd5ac60c8c0f80414c7f4b36862c/mpris_server-0.4.2.tar.gz' | tar ztf -
mpris_server-0.4.2/
mpris_server-0.4.2/LICENSE
mpris_server-0.4.2/MANIFEST.in
mpris_server-0.4.2/PKG-INFO
mpris_server-0.4.2/README.md
mpris_server-0.4.2/mpris_server/
mpris_server-0.4.2/mpris_server/__init__.py
mpris_server-0.4.2/mpris_server/adapters.py
mpris_server-0.4.2/mpris_server/base.py
mpris_server-0.4.2/mpris_server/events.py
mpris_server-0.4.2/mpris_server/interfaces/
mpris_server-0.4.2/mpris_server/interfaces/__init__.py
mpris_server-0.4.2/mpris_server/interfaces/interface.py
mpris_server-0.4.2/mpris_server/interfaces/player.py
mpris_server-0.4.2/mpris_server/interfaces/playlists.py
mpris_server-0.4.2/mpris_server/interfaces/root.py
mpris_server-0.4.2/mpris_server/interfaces/tracklist.py
mpris_server-0.4.2/mpris_server/mpris/
mpris_server-0.4.2/mpris_server/mpris/__init__.py
mpris_server-0.4.2/mpris_server/mpris/compat.py
mpris_server-0.4.2/mpris_server/mpris/metadata.py
mpris_server-0.4.2/mpris_server/server.py
mpris_server-0.4.2/mpris_server/types.py
mpris_server-0.4.2/mpris_server.egg-info/
mpris_server-0.4.2/mpris_server.egg-info/PKG-INFO
mpris_server-0.4.2/mpris_server.egg-info/SOURCES.txt
mpris_server-0.4.2/mpris_server.egg-info/dependency_links.txt
mpris_server-0.4.2/mpris_server.egg-info/requires.txt
mpris_server-0.4.2/mpris_server.egg-info/top_level.txt
mpris_server-0.4.2/mpris_server.egg-info/zip-safe
mpris_server-0.4.2/pyproject.toml
mpris_server-0.4.2/requirements.txt
mpris_server-0.4.2/setup.cfg
mpris_server-0.4.2/setup.py
```

**Comment by alexdelorenzo** [comment]
Thanks for opening an issue. 

Sure, I'll make sure `requirements.txt` gets distributed via PyPI. I'll ping you here when it's uploaded.

**Comment by alexdelorenzo** [comment]
@tomfitzhenry, `requirements.txt` is now included in the source distributions for the [`0.5.12`](https://pypi.org/project/aiopath/0.5.12/#files) and [`0.6.10`](https://pypi.org/project/aiopath/0.6.10/#files) releases on PyPI.

This should fix your issue. Let me know if there's anything I can do to make packaging easier on your end.

**Comment by alexdelorenzo** [comment]
I'm going to close this issue. Feel free to reopen it or create a new issue if this problem persists.
